lang_python.adoc

```diff
@@ -33,68 +33,94 @@ Runtime library API version must match compiler version that you're using
 +
-Reference: https://packages.debian.org/buster/python3-kaitaistruct[package info for Debian], https://packages.ubuntu.com/eoan/python3-kaitaistruct[package info for Ubuntu]
+Reference: https://packages.debian.org/buster/python-kaitaistruct[package info for Debian], https://packages.ubuntu.com/eoan/python-kaitaistruct[package info for Ubuntu]
+
 
-* If your project is using *requirements.txt* to manage your
-  dependencies, add the following line to it:
+* Add the following dependency specifier into
+`install_requires` of `options` section
+of your `setup.cfg` (or the arg of `setuptools.setup`
+with the same name in your `setup.py`) :
 +
 ----
 kaitaistruct
 ----
 +
-and then run `pip install -r requirements.txt` to update all your
-dependencies.
+and then run `pip install --upgrade -e ./` in the dir with the `setup.py`
+in order to install your package and all its dependencies in
+an editable way.
 
-* Otherwise, you can just install the package manually using *pip*
+* Otherwise, you can just install the package manually using `pip`
   (Python package manager):
 +
 [source,shell]
-pip install kaitaistruct
+pip install --upgrade kaitaistruct
 
 [[add-runtime-snapshot]]
-==== Using latest (snapshot) version
+==== Using latest (snapshot) version (RECOMMENDED)
 
 If you're using latest (unreleased AKA "unstable" AKA "snapshot") build
 of Kaitai Struct compiler, that will require latest runtime libraries as
 well:
 
-* If your project is using *requirements.txt* to manage your
-  dependencies, add the following line to it:
+* You can a make `pip` to use a bleeding edge version of the runtime by
+  adding a https://www.python.org/dev/peps/pep-0508/[PEP 508]
+  dependency specifier into `install_requires`:
 +
 ----
-kaitaistruct @ git+https://github.com/kaitai-io/kaitai_struct_python_runtime.git
+    kaitaistruct @ git+https://github.com/kaitai-io/kaitai_struct_python_runtime.git
 ----
 +
-and then run `pip install -r requirements.txt` to update all your
-dependencies.
+.
 
 * Otherwise, you can just install the package manually using *pip*
   (Python package manager):
 +
 [source,shell]
 pip install --upgrade --pre git+https://github.com/kaitai-io/kaitai_struct_python_runtime.git
 
+[[secure-installation]]
+==== More secure systemwide installation (RECOMMENDED)
+
+Since you usually don't want to run any builds from root, you can do
+the following if you are sure that there are no malware that can
+compromise the content of the build dir:
+
+```sh
+# run unelevated:
+git clone https://github.com/kaitai-io/kaitai_struct_python_runtime.git
+cd kaitai_struct_python_runtime
+python3 ./setup.py bdist_wheel
+# inspect the content of the zip archive in ./dist
+
+# run elevated:
+sudo pip3 install --upgrade ./dist/*.whl
+```
+
+This is secure only because wheels cannot
+execute commands during installation.
+Though you still need to check what you install.
+
 [[add-dependencies]]
-=== Adding dependencies
+=== Dealing with old python versions
```

Comment:
Please don't rename these sections, especially without changing the anchor. The idea is to have uniform set of sections, the same ones for all languages.

Reply:
> The idea is to have uniform set of sections, the same ones for all languages.

That part isn't obvious IMO - maybe it should be documented somewhere? Not sure what would be the best place for that kind of information though - maybe a short comment at the top of every `lang_*.adoc` file?

Reply:
Should I change the anchor or should I revert the rename?